utils/src/main/java/com/cloud/utils/net/NetUtils.java

```diff
@@ -487,6 +490,14 @@ public static boolean validIpRange(final String startIP, final String endIP) {
         return startIPLong <= endIPLong;
     }
 
+    public static boolean isValidMtu(final Integer mtu) {
```

Comment:
Looks good!

One thing though, I don't see a UnitTest for this method.

Could you just add a UnitTest to test it? Then it looks good to go!

Reply:
Good hint, I added a test and hope you are fully satisfied :)